qa/backwards/shared/src/main/java/org/elasticsearch/test/ExternalNodeService.java

```diff
@@ -369,55 +348,52 @@ private void start(Deque<String> commandLine) throws IOException {
                 message("process forked");
-
-                Matcher m = readUntilMatches(stdout, "pid", ".+\\[node.+ pid\\[(\\d+)\\].*", timeValueSeconds(10));
-                pid = m.group(1);
-                message("pid is [" + pid + "]");
-                m = readUntilMatches(stdout, "transport address", ".+\\[transport .+bound_addresses .*\\{(?:127\\.0\\.0\\.1|\\[::1\\]):(\\d+)\\}.*",
-                        timeValueSeconds(20));
-                port = m.group(1);
-                message("bound to [localhost:" + port + "]");
-                readUntilMatches(stdout, "started", ".+\\] started$", timeValueSeconds(20));
-                runningElasticsearches.put(port, new ProcessInfo(process, pid));
+                runningElasticsearches.put(pid, new ProcessInfo(process, pid));
+                message("pid [" + pid + "]");
                 message("started");
-                process.getInputStream().close();
             } finally {
-                if (process != null && (port == null || !runningElasticsearches.containsKey(port))) {
-                    logger.error("It looks like we failed to launch elasticsearch. Kill -9ing it just to make sure it doesn't linger.");
+                if (pid == null) {
+                    logger.error("It looks like we failed to launch elasticsearch.");
                     logger.error("We tried to start it like this: {}", startReproduction);
-                    // In Java 1.8 this should be destroyForcibly
-                    process.destroy();
-                    if (pid != null) {
-                        try {
-                            new ProcessInfo(null, pid).stop();
-                        } catch (InterruptedException e) {
-                            logger.warn("Failed to stop busted elasticsearch at [{}]", pid);
-                        }
+                    if (process != null) {
+                        // In Java 1.8 this should be destroyForcibly
+                        process.destroy();
                     }
-                    if (stdout != null) {
-                        try {
-                            stdout.reset();
-                            CharBuffer lines = CharBuffer.allocate(1024 * 1024);
-                            stdout.read(lines);
-                            lines.flip();
-                            logger.error(
-                                    "We were able to capture some of elasticsearch's output. Hopefully this will be useful in figuring out the failure:\n{}",
-                                    lines.toString());
-                        } catch (IOException e) {
-                            logger.warn("Sadly we couldn't capture any of its output.", e);
-                        }
+                }
+            }
+        }
+
+        private String getPid(String pidFile) throws IOException {
+            Path pidPath = getPath(pidFile).toAbsolutePath();
+            long maxTimeInMillis = 10000;
```

Comment:
increased the timeout here: https://github.com/brwe/elasticsearch/blob/external-nodes-inherit-io/qa/backwards/shared/src/test/java/org/elasticsearch/test/CompositeTestCluster.java#L87